client/lbclient/boot/replication.js

```diff
@@ -23,17 +23,33 @@ module.exports = function(client) {
   };
 
   // setup model replication
-  function sync(cb) {
-    if (client.network.isConnected) {
-      RemoteTodo.replicate(LocalTodo, function() {
-        LocalTodo.replicate(RemoteTodo, cb);
+  var since = {push: -1, pull: -1};
+  function sync() {
+    LocalTodo.replicate(
+      RemoteTodo,
+      since.push,
+      function pushed(err, conflicts, cps) {
+        console.log('LR', arguments);
```

Comment:
Is the URL "/api/Todos/checkpoint" available in the explorer?

Please create a new RemoteTodo model per the instructions in [docs](http://docs.strongloop.com/display/LB/Synchronization#Synchronization-Createaclientapp) and use it instead of the base Todo model here.

```diff
- var RemoteTodo = client.models.Todo;
+ var RemoteTodo = client.models.RemoteTodo;
```